### DISCUSSION TITLE
What is the difference between prompt and initial_prompt

### DISCUSSION THREAD

**Discussion by rv-outplay** [openai/whisper#1189] [General]
I wanted to set some context to the model, I used the same prompt in "prompt" param and then "initial_prompt" param, not sure why the results are better when I use "initial_prompt". 

What is the difference between both and when can what be used ? Or can they be used interchangeably.

**Comment by Tex2002ans** [comment]
> What is the difference between prompt and initial_prompt [...] and when can what be used?

The difference between `prompt` + `initial_prompt` was answered by jongwook in previous topics.

On `initial_prompt`, see:

- https://github.com/openai/whisper/discussions/963#discussioncomment-4987057

On `prompt`, see:

- https://github.com/openai/whisper/discussions/1080

which pointed to his much more detailed `prompt` answer here:

- https://github.com/openai/whisper/discussions/117#discussioncomment-3727051
